include/crypto_config.h

```diff
@@ -160,7 +160,7 @@
 
 // TC Defines
 #define TC_SH_SIZE 8 /* bits */
-#define TC_SN_SIZE 0
+#define TC_SN_SIZE 4
```

Comment:
These defines are only used for max sizes right? Are 4 byte SN's the max allowed per the blue book?